src/utils/intrusive_ptr_test.cc

```diff
@@ -46,6 +46,11 @@ class RefCountedType {
   unsigned int count_;
 };
 
+class RefCountedTypeDerived: public RefCountedType {
+ public:
+  RefCountedTypeDerived() {}
+};
```

Comment:
I _think_ you can remove the constructor: 

```suggestion
class RefCountedTypeDerived: public RefCountedType {};
```